packages/expect/__typechecks__/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "skipLibCheck": true
```

Comment:
Simply for better performance. `tsc` will do this check anyway, no need to run it with each test.